include/main.h

```diff
@@ -0,0 +1,12 @@
+#ifndef GUARD_MAIN_H
+#define GUARD_MAIN_H
+
+typedef void (*IntrCallback)(void);
+
+extern EWRAM_DATA IntrCallback gIntrCallbacks[6];
```

Comment:
```suggestion
extern IntrCallback gIntrCallbacks[6];
```

EWRAM_DATA is just a hint for the linker to put it in the right section in the `.c` file. Since this is a header, just `extern` is fine.

Reply:
Removed